### ISSUE TITLE
va-loading-indicator - review usage of aria-valuetext

### ISSUE THREAD

**Issue by powellkerry** [department-of-veterans-affairs/vets-design-system-documentation#4205]
## Findings details
The message prop sets the `aria-valuetext` attribute but there is no `aria-valuenow` attribute set.

`aria-valuetext` is used in conjunction with `aria-valuenow` to convey the value in a human readable format. It is not recommended to use `aria-valuetext` without `aria-valuenow`. See [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-valuetext) docs for further reading.

The message prop sets the `aria-valuetext` attribute on the component, and the label sets the `aria-label` on the component. In an AI query to find all instances on vets-website where a message was defined it looks like all except for 1 do not set both a label and a message. The instance that does set both uses different phrasing to convey the same message. 

### Must
Remove usage of `aria-valuetext` attribute.

### Recommendations: 
@department-of-veterans-affairs/platform-design-system-a11y should be consulted to find the correct attributes to use. @jeana-adhoc suggested that we simplify the component and just use the `alt` attribute. We could also just use `aria-label` and add in logic to determine if the label or message should be used for that attribute.

### Acceptance Criteria
All props are read aloud by screen-reader

## Type of issue
- [ ] Code review
- [ ] Readability
- [ ] Automated scans
- [ ] Use of color
- [ ] Text resizing, zoom, and magnification
- [ x] Screen readers
- [ ] Input and interaction methods

**Comment by caw310** [comment]
Moving to sprint 27 as no one was able to work on this.
